docs/user/check-ref/redis_wrapper.rst

```diff
@@ -10,6 +10,16 @@ Read-only access to Redis servers is provided by the :py:func:`redis` function.
     of the current entity's ``host`` attribute, and :samp:`{<port>}` is the given port (default ``6379``). See
     below for a list of methods provided by the returned connection object.
 
+    :param host: Redis host.
+    :type host: str
+
+    :param password:  If set - enables authentication to the destination server with the password provided.
```

Comment:
Please add `Default is None`, to avoid confusion for non-auth redis servers.

Reply:
added